### ISSUE TITLE
pane resize handles overflow over editors

### ISSUE THREAD

**Issue by braver** [atom/atom#7446]
If you have a vertical split, the resize handle overflows to the right, possibly over an adjacent editor. This messes with drag-select. I haven't found out how to solve this. 

The layout to reproduce this:
```
---------------------
|         |         |
+---------|         |
|         |         |
---------------------
```

cc @simurai 

![screen shot 2015-06-26 at 10 52 04](https://cloud.githubusercontent.com/assets/2543659/8373997/fff9d38e-1bf1-11e5-92d5-c23ddedaa700.png)

**Comment by izuzak** [comment]
Strange. I'm having trouble reproducing this:


![screen shot 2015-06-26 at 13 04 19](https://cloud.githubusercontent.com/assets/38924/8376161/47e41044-1c04-11e5-8732-774498044717.png)

Does it happen every time, and can you list the exact steps you're performing after starting Atom? Also, can you provide [other information](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#submitting-issues) and clarify which themes you're using?

**Comment by braver** [comment]
It happens all the time, also in safe mode.

Steps:
open a file
split it to the right
split the left editor to the bottom

Tested in One Dark, Atom Dark and Isotope-UI.

**Comment by izuzak** [comment]
How about some OS and Atom version info? :smile:

**Comment by braver** [comment]
Oh, sorry man, my focus is not there today.

OSX 10.10.4, Atom 1.0.0

**Comment by simurai** [comment]
Also having trouble to reproduce. From the screenshot, it looks like scrollbars are always enabled. Does that have an influence?

**Comment by braver** [comment]
Good catch. I'm now without external mouse, so my Mac switches to no-scrollbars, and the problem has gone away.

**Comment by simurai** [comment]
I actually can't reproduce it even when I turn the scrollbars on. Maybe another factor: Are you on a non-retina screen? Currently I only can test on my retina MB.

**Comment by braver** [comment]
Well, as you can imagine, I cannot reproduce it either. I have been able to do so earlier today on two installations and now it's gone.
I love computers...

**Comment by simurai** [comment]
Yup, computers sometimes can be quite the mystery boxes.

**Comment by mnquintana** [comment]
I got something like this once recently with a horizontal split - I haven't figured out how to reproduce it though. :confounded:

**Comment by ThommyTheThird** [comment]
Since this was opened as an OSX issue, I'd like to add some information as I've encountered it on Windows.
Atom 1.0.2, Windows 7, x64

Reproduced:
- have one pane open with two files
- Split file down (should be 2 files top and 1 bottom now)
- Split that same top file to the right
![atom-io-panehandle-bug](https://cloud.githubusercontent.com/assets/4256041/8908140/9778db88-347a-11e5-9c8f-cdf10e648d0a.png)

**Comment by NuclearCookie** [comment]
I have the same issue on windows:
Atom 1.0.3, Windows 8 x64. On my machine it works with the reproduction given above. On another pc running ArchLinux, I cannot reproduce the issue.

**Comment by 50Wliu** [comment]
I've been able to reproduce this on Atom 1.9.0-dev-9175194, Windows 10 x64.

**Comment by JetLua** [comment]
I've been able to reproduce this on Atom 1.11.0-beta0, Windows 7 x64.
---

![1](https://cloud.githubusercontent.com/assets/6738986/18154380/adadcae6-7036-11e6-9017-dec8484dc2b4.gif)

**Comment by frantic1048** [comment]
![screenshot_20160901_211901](https://cloud.githubusercontent.com/assets/3102803/18168764/deca4034-7089-11e6-8c86-a2ff80646b51.png)

Same issue here:

Steps:

1. open a new tab
2. split current tab on vertical direction
3. split the tab **above** on horizontal direction

now the layout becomes:

```plain
+--------------+------------+
|              |            |
|              |            |
|              |            |
|              |            |
+--------------+------------+
|                           |
|       <bottom tab>        |
|                           |
+---------------------------+
```

The `atom-pane-resize-handle` between two above tabs overflows into **bottom tab**, what the point is it's on the top of bottom tab(as the first screenshot shows), thus become operable on invisible area.

Version Info:
OS: Arch Linux latest
Atom    : 1.9.9
Electron: 0.37.8
Chrome  : 49.0.2623.75
Node    : 5.10.0

I found a quick fix is to remove https://github.com/atom/atom/blob/master/static/panes.less#L19 `z-index: 3;` for `<atom-pane-resize-handle>`.

I looked over the styles, but not found a resonable demand setting `z-index` here.

**Comment by frantic1048** [comment]
After looked deep into the overflowed `atom-pane-resize-handle`(`the handle`) element, I find it uses `height: 100%` gained its further parent's height, results the scrrenshot on the previous comment: the highlighted handle gains the upper right pane's height.

I just made a PR make `the handle` inherit the correct height. ヾ(⌒(\_*'ω'*)\_

**Comment by simurai** [comment]
I can reproduce this now too and @frantic1048's fix https://github.com/atom/atom/pull/12580 seems to work fine. Thanks again. It should be fixed in Atom `1.12`.

If somebody wants, you can already try it out (and see if there are any regressions) by adding this to your `styles.less` file:

```css
atom-pane-axis {
  position: relative;
}
```
